Scripts/DatabankLib/analyze.py

```diff
@@ -625,171 +659,326 @@ def computeFF(  # noqa: N802 (API)
-                        f"echo System |  {trjconv_cmd} -f {trj_name} "
-                        f"-s {tpr_name} -o {xtcwhole} -pbc mol -b {str(eq_time)}"
-                    )
+                    os.system(f'echo System |  {trjconv_cmd} -f {trj_name} '
+                              f'-s {tpr_name} -o {xtcwhole} -pbc mol -b {str(eq_time)}')
 
-                if not os.path.isfile(xtcfoo):
-                    os.system(
-                        f'echo "centralAtom\nSystem" |  {trjconv_cmd} -center'
-                        f" -pbc mol -n foo.ndx -f {xtcwhole} -s {tpr_name}"
-                        f" -o {xtcfoo}"
-                    )
+                if (not os.path.isfile(xtcfoo)):
+                    os.system(f'echo "centralAtom\nSystem" |  {trjconv_cmd} -center'
+                              f' -pbc mol -n foo.ndx -f {xtcwhole} -s {tpr_name}'
+                              f' -o {xtcfoo}')
 
-                os.system("rm foo.ndx")
-                os.system("rm " + xtcwhole)
+                os.system('rm foo.ndx')
+                os.system('rm ' + xtcwhole)
 
                 # Center around the center of mass of all the g_3 carbons
                 # if (not os.path.isfile(xtccentered)):
-                os.system(
-                    f'echo "a {g3_atom}\nq" | {makendx_cmd}'
-                    f" -f {tpr_name} -o foo.ndx"
-                )
-                os.system(
-                    f'echo "{g3_atom}\nSystem" |  {trjconv_cmd} -center'
-                    f" -pbc mol -n foo.ndx -f {xtcfoo} -s {tpr_name}"
-                    f" -o {xtccentered}"
-                )
-                os.system("rm " + xtcfoo)
-                os.system("rm foo.ndx")
+                os.system(f'echo "a {g3_atom}\nq" | {makendx_cmd}'
+                          f' -f {tpr_name} -o foo.ndx')
+                os.system(f'echo "{g3_atom}\nSystem" |  {trjconv_cmd} -center'
+                          f' -pbc mol -n foo.ndx -f {xtcfoo} -s {tpr_name}'
+                          f' -o {xtccentered}')
+                os.system('rm ' + xtcfoo)
         else:
-            print(
-                "Centering for other than Gromacs may not work if there are"
-                " jumps over periodic boundary conditions in z-direction."
+            print("Centering for other than Gromacs may not work if there are"
+                  " jumps over periodic boundary conditions in z-direction.")
+
+        if 'gromacs' in system['SOFTWARE']:
+            u = mda.Universe(tpr_name, xtccentered)
+        if 'openMM' in system['SOFTWARE'] or 'NAMD' in system['SOFTWARE']:
+            u = mda.Universe(struc_name, trj_name)
+
+        # -- PHILIP code starts here --
+        # We us a hardoced bin width
+        bin_width = 0.3
+
+        # Guesser might charged elements wrong like Cl- may guessed as Cl...
+        from DatabankLib.settings import elements
+        u.guess_TopologyAttrs(force_guess=["elements"])
+        elements.guess_elements(system, u)
```

Comment:
I think we can remove the MDA guesser since our own guesser does the same job.

```suggestion
        from DatabankLib.settings import elements
        elements.guess_elements(system, u)
```